File: c/1160.c
#include <stdio.h>
#include <stdbool.h>
 
int main() {
    int t;
    int pa;
    int pb;
    double g1;
    double g2;
    int years;
    
    scanf("%d", &t);
    
    for (int i = 0; i < t; i++) {
        years = 0;
        scanf("%d %d %lf %lf", &pa, &pb, &g1, &g2);
        
        g1 = g1 / 100;
        g2 = g2 / 100;
         
        while (true) {
            pa = pa + (g1 * pa);
            pb = pb + (g2 * pb);
            years++;
            
            if (years > 100 || pa > pb) {
                break;
            }
        }
        
        if (years > 100) {
            printf("Mais de 1 seculo.\n");
        } else {
            printf("%d anos.\n", years);
        }
    }
 
    return 0;
}

File: src/python/1160.py
t = int(input())

for i in range(t):
    years = 0
    values = input().split(' ')
    pa, pb, g1, g2 = values
    pa = int(pa)
    pb = int(pb)
    g1 = float(g1)
    g2 = float(g2)
    
    g1 = g1 / 100
    g2 = g2 / 100

    while (True):
        pa = int(pa + (g1 * pa))
        pb = int(pb + (g2 * pb))
        years += 1
        
        if (years > 100 or pa > pb):
            break

    if (years > 100):
        print('Mais de 1 seculo.')
    else:
        print('{} anos.'.format(years))